doc/langdef.md

```diff
@@ -942,31 +942,31 @@ specified by the following overloads:
       size
     </th>
     <td>
-      (string) -> int
+      string.() -> int
```

Comment:
Hmm, I'm okay marking the function-style version as "deprecated", but I don't think it's okay to just remove it. Anyone attempting to implement a compatible CEL implementation will need to provide the function-style versions for the foreseeable future, so it ought to be in the language spec.

Reply:
Done.

Reply:
Ugh, now it's a confusing example because we're not listing the full set of overloads. Unfortunately, the other functions with multiple overloads are either:

- Operators, which introduce an unnecessary complication.
- Conversion functions, which we single out in their description as being different.
- The `get*()` functions on timestamps and durations, but we're going to deprecate them on the durations.
- Other oddballs like `dyn()`.

So consider whether there's a better example to use here, but I'm okay with just the receiver-style size functions if you don't think there's a better alternative.